lib/ansible/modules/windows/win_chocolatey.ps1

```diff
@@ -204,6 +209,11 @@ Function Choco-Upgrade
         $cmd += " -ignoredependencies"
```

Comment:
Yeah, so I think they started doing it differently in newer releases. But we broke win_chocolatey as well by introducing the `--exact` option. This fails to work for chocolatey older than 0.9.10 :-(

Not sure if we want to fix that (by checking the version ourselves), but if we don't, we should update the docs to state chocolatey 0.9.10+ is required.